lib/ansible/modules/network/f5/bigip_command.py

```diff
@@ -0,0 +1,379 @@
+    'metadata_version': '1.0'
+}
+
+DOCUMENTATION = '''
+---
+module: bigip_command
+short_description: Run arbitrary command on F5 devices.
+description:
+  - Sends an arbitrary command to an BIG-IP node and returns the results
+    read from the device. This module includes an argument that will cause
+    the module to wait for a specific condition before returning or timing
+    out if the condition is not met.
+version_added: "2.4"
+options:
+  commands:
+    description:
+      - The commands to send to the remote BIG-IP device over the
+        configured provider. The resulting output from the command
+        is returned. If the I(wait_for) argument is provided, the
+        module is not returned until the condition is satisfied or
+        the number of retires as expired.
+      - The I(commands) argument also accepts an alternative form
+        that allows for complex values that specify the command
+        to run and the output format to return. This can be done
+        on a command by command basis. The complex argument supports
+        the keywords C(command) and C(output) where C(command) is the
+        command to run and C(output) is 'text' or 'one-line'.
+    required: True
+  wait_for:
+    description:
+      - Specifies what to evaluate from the output of the command
+        and what conditionals to apply.  This argument will cause
+        the task to wait for a particular conditional to be true
+        before moving forward. If the conditional is not true
+        by the configured retries, the task fails. See examples.
+    required: False
+    default: None
+    aliases: ['waitfor']
+    version_added: "2.2"
+  match:
+    description:
+      - The I(match) argument is used in conjunction with the
+        I(wait_for) argument to specify the match policy. Valid
+        values are C(all) or C(any). If the value is set to C(all)
+        then all conditionals in the I(wait_for) must be satisfied. If
+        the value is set to C(any) then only one of the values must be
+        satisfied.
+    required: False
+    default: all
+    version_added: "2.2"
+  retries:
+    description:
+      - Specifies the number of retries a command should by tried
+        before it is considered failed. The command is run on the
+        target device every retry and evaluated against the I(wait_for)
+        conditionals.
+    required: False
+    default: 10
+  interval:
+    description:
+      - Configures the interval in seconds to wait between retries
+        of the command. If the command does not pass the specified
+        conditional, the interval indicates how to long to wait before
+        trying the command again.
+    required: False
+    default: 1
+notes:
+  - Requires the f5-sdk Python package on the host. This is as easy as pip
+    install f5-sdk.
+  - Requires Ansible >= 2.3.
+requirements:
+  - f5-sdk >= 2.2.3
+extends_documentation_fragment: f5
+author:
+  - Tim Rupp (@caphrim007)
```

Comment:
@Akasurde has this become a new requirement? generally I haven't used quotes across any of the documentation variable strings unless there's an errant symbol that causes it to not parse correctly. I reserve quotes for values in the Example doc block as that's what I encourage in practice as I have seen cases where an errant symbol causes a parsing error